src/commoncode/codec.py

```diff
@@ -26,13 +26,22 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+
+from commoncode.system import py2
+
 """
 Numbers to bytes or strings and URLs coder/decoders.
 """
 
 padding = b'/'
 
 b85_symbols = b'0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz!#$%&()*+-;<=>?@^_`{|}~'
+
+if not py2:
+    # bytes are no longer "strings" on Python3, but an array of integers not characters
+    # FIXME: there is likely a better way
+    b85_symbols = tuple(bytes(c, encoding='ascii') for c in b85_symbols.decode('ascii'))
```

Comment:
I see the comment on line 42,
I tried around some variations with map and lambda to avoid using the list comprehension.
We can use this instead:
```
b85_symbols = tuple(map(lambda x : bytes(chr(x), encoding='ascii'), tuple(b85_symbols)))
```

I'll keep looking for more elegant solution.
cc: @pombredanne